packages/connectivity-tests/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "connectivity-tests",
+  "name": "@mongosh/connectivity-tests",
```

Comment:
Shouldn’t have any effect at all, but matches our other packages (including the private ones) and avoids a potential naming conflict